addons/website_slides/static/src/js/slides_embed.js

```diff
@@ -1,5 +1,50 @@
 /*global $, _, PDFSlidesViewer*/
+odoo.define('website_slides.slides_embed', function (require) {
```

Comment:
Are you sure about moving this in this file without any adaptation ? The file is not in the same asset bundle...